features/bevragen/persoon/verblijfplaats/locatie/protocollering-antwoord.feature

```diff
@@ -0,0 +1,39 @@
+#language: nl
+
+Functionaliteit: protocollering van het antwoord voor verblijfplaats locatie
+
```

Comment:
Zie opmerking in regel 4 van features/bevragen/persoon/verblijfplaats/onbekend/protocollering-antwoord.feature.